third_party/VENDOR-LICENSE

```diff
@@ -1,8161 +0,0 @@
-
```

Comment:
I'm curious - why did this file change as part of this PR?  Result of an automated process?

Reply:
It’s result of `./hack/update-codegen.sh`.